src/rules/converters/no-unused-expression.ts

```diff
@@ -0,0 +1,32 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoUnusedExpression: RuleConverter = tslintRule => {
+    return {
+        rules: [
+            {
+                ruleName: "no-unused-expressions",
+                ...collectNoticesAndArguments(tslintRule.ruleArguments),
+            },
+        ],
+    };
+};
+
+const collectNoticesAndArguments = (tsLintRuleArguments: any[]) => {
+    if (tsLintRuleArguments.length === 0) {
+        return undefined;
+    }
+
+    const notices = [];
+    const ruleArguments: any[] = [];
+
+    if (tsLintRuleArguments.includes("allow-tagged-template")) {
+        ruleArguments.push({ allowTaggedTemplates: true });
+    } else {
+        notices.push(`ESLint does not support optional config ${tsLintRuleArguments[0]}.`);
+    }

```

Comment:
Hmm, I don't think this is quite right. Consider the following possibilities for `tslintRuleArguments`:

* `[]`: This will put `undefined` in the notice string
* `["allow-fast-null-checks"]`: The equivalent ESLint option is `"allowShortCircuit"`
* `["allow-new"]`: There should be no notice here, as this is the default ESLint behavior

Note that because the optional `allow-new` argument in TSLint is the default ESLint behavior, there should be a notice if the TSLint rule arguments _don't_ include `allow-new`.

References: [TSLint's `no-unused-expression`](https://palantir.github.io/tslint/rules/no-unused-expression/) / [ESLint's `no-unused-expression`](https://eslint.org/docs/rules/no-unused-expressions)

Reply:
Hi! I added the converter for allow-fast-null-checks and notice for TSLint rule without allow-new config.